app/src/main/java/org/walleth/activities/SelectTokenActivity.kt

```diff
@@ -56,6 +57,20 @@ class SelectTokenActivity : AppCompatActivity() {
         fab.setOnClickListener {
             startActivityFromClass(CreateTokenDefinitionActivity::class)
         }
+        search.setOnQueryTextListener(object : SearchView.OnQueryTextListener {
+            override fun onQueryTextSubmit(query: String?): Boolean {
```

Comment:
nit: like `override fun onQueryTextSubmit(query: String?) = true ` a bit more and it is the style used in WALLETH - or are you opposed to this?